app/components/agenda/side-nav/agenda.hbs

```diff
@@ -1,7 +1,7 @@
 <Auk::TabExpanded
   data-test-agenda-side-nav-agenda
   @route="agenda.agendaitems"
-  @current-when="agenda.agendaitems"
+  @current-when={{(eq @currentAgenda.id @agenda.id)}} {{!-- as by LinkTo specs, @current-when can also be a boolean, which is needed here --}}
```

Comment:
Reverted this to how it's being done on `dev` as `agenda.agendaitems` doesn't cover all bases.